pkg/server/sqlflowserver_test.go

```diff
@@ -125,12 +125,13 @@ func TestSQL(t *testing.T) {
 	_, err = stream.Recv()
 	a.Equal(status.Error(codes.Unknown, "Lex: Unknown problem ..."), err)
 
-	testMultipleSQL := fmt.Sprintf("%s %s", testQuerySQL, testExtendedSQL)
```

Comment:
@typhoonzero I don't think we need to test multiple SQL in the unit test since we are using `mockRun`.